fibonacci.js

```diff
@@ -0,0 +1,12 @@
+function firstNFibonacciNumbers(n) {
+    let firstTerm = 0, secondTerm = 1, nextTerm = 0;
+    for (let i = 1; i <= n; i++) {
+        console.log(firstTerm);
+        nextTerm = firstTerm + secondTerm;
+        firstTerm = secondTerm;
+        secondTerm = nextTerm;
+    }
+
```

Comment:
It should return an array. Currently it is returning undefined. Please fix this.